scalatest/src/main/scala/org/scalatestplus/package.scala

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2001-2013 Artima, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.scalatestplus
+
+// SKIP-SCALATESTJS-START
+/**
+ * Package objects of type alias for deprecated easymock classes, there will be moved in these new packages in the future version of ScalaTest.
+ */
+package object easymock {
+
+  type EasyMockSugar = org.scalatest.easymock.EasyMockSugar
+
+  lazy val EasyMockSugar = org.scalatest.easymock.EasyMockSugar
+
+}
+
+/**
+ * Package objects of type alias for deprecated mockito classes, there will be moved in these new packages in the future version of ScalaTest.
+ */
+package object mockito {
+
+  type MockitoSugar = org.scalatest.mockito.MockitoSugar
+
+  lazy val MockitoSugar = org.scalatest.mockito.MockitoSugar
```

Comment:
@cheeseng Any reason these are lazy vals instead of just plain old vals?

Reply:
@cheeseng Not special reason, just get used to lazy val already for non-local values. :)